src/tss2-tcti/tctildr-dl.c

```diff
@@ -56,6 +56,10 @@ struct {
         .file = "libtss2-tcti-mssim.so.0",
         .description = "Access to libtss2-tcti-mssim.so",
     },
+    {
+        .file = "libtss2-tcti-libtpms.so.0",
```

Comment:
IMHO, we should not add libtpms to the autoloading logic of tctildr.

Otherwise, you'll never detect a badly setup system, which can be kind of surprising. I would want to keep the current behavior where the tss fails if you messed up.

Reply:
Agreed. This would lead to the TSS failing in weird ways when there is no real TPM but libtpms installed (_"Why has my tpm2_startup returned success but now I get error 0x100 not initialized"_).